src/react_native/async_storage.cljs

```diff
@@ -0,0 +1,14 @@
+(ns react-native.async-storage
+  (:require ["@react-native-async-storage/async-storage" :default AsyncStorage]))
+
+
+(defn set-item
+  [key value]
+  (-> (.setItem AsyncStorage (clj->js key) (str value))
+      (.catch #(js/console.error (str "Error storing data in async-storage: " %)))))
+
+(defn get-item
+  [key callback]
+  (-> (.getItem AsyncStorage (clj->js key))
+      (.then #(callback %))
+      (.catch #(js/console.error (str "Error retrieving data from async-storage: " %)))))
```

Comment:
Good question @ibrkhalil. Essentially I think your question boils down to whether to use a logging library or not. There are many reasons to prefer a log library over a simple debugging function like `js/console.error`.

- The log library is optimized for raw performance.
- We can control verbosity by using multiple log levels. Just call `(log/set-level! :debug)` in your REPL and you'll start to see all logs in the mobile app.
- We can define filter rules. Not really used yet in mobile afaik, but this can be handy.
- We can define custom formatters and multiple log *appenders* to output to destinations different than `STDOUT`. For instance, I used a custom `timbre` appender for capturing logs in a vector of strings for use in unit tests. See https://github.com/status-im/status-mobile/blob/d902fb10d1f53f6ecc32ef594336db1c10152b32/src/test_helpers/unit.cljs#L114

And probably many other advantages.